README.md

```diff
@@ -34,87 +79,69 @@ goe wallet unlock
 goe wallet lock
 ```
 
-**🔑 Note: Other wallet-dependent operations require the wallet to be unlocked.**
-
+> 🔑 **Note:** Wallets are secured using a password-derived key stored in your system keychain.
+> - **Unlock**: Enter your password to derive a key and store it in the system keychain to decrypt your private key for Git operations.
+> - **Lock**: Remove the derived key from the system keychain to secure your wallet.
 
-## 2. Repo CLI
 
-The repo CLI allows you to create repositories, set branches, and manage permissions on-chain.
+## 2. Repo Command
 
-### Commands
+Create and manage on-chain repositories and permissions.
 
+- **Create a repository**
+```bash
+goe repo create <repo_name> --chain-id <chain_id>
+```
 
 - **List repositories**
 ```bash
 goe repo list --chain-id <chain_id>
-
-# eg.
-goe repo list --chain-id  11155111
 ```
 
-
-- **Create a repository**
+- **List branches**
 ```bash
-goe repo create <repo_name> --chain-id <chain_id>
-
-# eg.
-goe repo create test-repo --chain-id 11155111
+goe repo branches <repo_address> --chain-id <chain_id>
 ```
 
 - **Set default branch**
 ```bash
 goe repo default-branch <repo_address> <branch_name> --chain-id <chain_id>
-
-# eg.
-goe repo default-branch 0x0533dc9CD8...aF4279Bda20f55 master --chain-id 11155111
 ```
 
-
-- **Grant push access**
+- **Grant / Revoke push access**
 ```bash
-goe repo grant-push <repo_address> <puser_address> --chain-id <chain_id>
-
-# eg.
-goe repo grant-push 0x0533dc9CD84D...dfaF4279Bda20f55 0x1234... --chain-id 11155111
-```
-
-- **Revoke push access**
-```bash
-goe repo revoke-push <repo_address> <puser_address> --chain-id <chain_id>
-
-# eg.
-goe repo revoke-push 0x0533dc9CD84D...dfaF4279Bda20f55 0x1234... --chain-id 11155111
+goe repo grant-push <repo_address> <user_address> --chain-id <chain_id>
```

Comment:
It would be nice if we could set the default chain ID rather than specifying it in every command, as it should not change frequently.